### PR TITLE
FE/UI tutorials refined

### PR DESCRIPTION
**Pull request by Mingela**
- Squashed original commit history by @tonysani 
- Added adjustments by @yamkovoy 
- Repaired branch to merge to `main`

### CODE REVIEW

**Comment by yamkovoy** [comment]
### Changes
This PR introduces the following changes:
- Adapted the original FIB Web UI documentation by @tonysani into the following new topics: **Overview** > **Web UI** topic; **Tutorials: Web UI** section. 
- Added new **Tutorials: API** > **Signing user email addresses** topic to address the following issue: #34.
- Adapted the rest of the documentation to address the addition of new content (e.g., renamed **Tutorials** to **Tutorials: API**, added additional links throughout documentation, etc.).
- Minor wording and formatting changes throughout the documentation.

---

### Thoughts on screenshots
For the time being, I have removed the screenshots that were originally uploaded with the FIB Web UI documentation by @tonysani.

As to how to proceed regarding the presence of the Web UI screenshots, I suggest the following:
1. Add a single spoilered screenshot per Web UI page entry in the **Overview** > **Web UI** topic.
2. Add a single spoilered `Example` screenshot per tutorial entry in the **Tutorials: Web UI** section.

@tonysani and @Mingela please let me know your thoughts on the matter in the following issue: #36.

File: src/overview/web-interface.md [hunk 1]
```diff
@@ -40,17 +42,16 @@ On this page peers can view their current balance.
...
 
 On this page peers can view the following information about their account:
+
 - **About** tab — displays the peer's contact information.
 - **Subscription** tab — displays the following subscription information:
-    - **Membership type**: *?*
-    - **Subscription type**: the type of the subscription that a peer has.
-    - **Company name**: the name of the company that a peer is registered with.
-    - **Operating country**: the country that a peer is operating from.
+  - **Membership type**: _?_
```
Comment (Mingela) [inline src/overview/web-interface.md:48]: I guess it's FREEMIUM/PREMIUM

File: src/overview/web-interface.md [hunk 2]
```diff
@@ -40,17 +42,16 @@ On this page peers can view their current balance.
...
+  - **Subscription type**: the type of the subscription that a peer has.
+  - **Company name**: the name of the company that a peer is registered with.
+  - **Operating country**: the country that a peer is operating from.
 - **Services** tab — displays whether a peer has access to the following FIB services:
-    - **Uploading**: allows users to upload fraud events data.
-    - **?**: *?*
-    - **Reports**: allows users to access the fraud analytics reports.
+  - **Uploading**: allows users to upload fraud events data.
+  - **?**: _?_
```
Comment (Mingela) [inline src/overview/web-interface.md:54]: @tonysani please advise what's the service missing here

File: src/tutorials-api/signing-user-email-addresses.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Every API request to [authorize a user in the system](../api-specification/auth-controller/authorizing-a-user-in-the-system.md) requires a signature of the user's email address.
+
+The exact implementation to your system may vary depending on the specific SDK or programming language being used. However, the overall sequence of events to sign a user email address should be the following:
+
+1. Create a `keyPair` object from the public and private keys of the user's authentication key pair.
+2. Obtain a `signature` of the user's email address.
+3. Encode the `signature` as a Hex string.
+
+The resulting encoded `signature` Hex string can be used as the body for requests to the following endpoint:
```
Comment (Mingela) [inline src/tutorials-api/signing-user-email-addresses.md:11]:
```suggestion
The resulting encoded `signature` Hex string can be used as a part of the body for requests to the following endpoint:
```

**Comment by Mingela** [comment]
Looks good to me

File: src/tutorials-web.md
Comment (tonysani) [inline src/tutorials-web.md:1]: I would suggest changing the order to uploading, flagging and downloading as a natural sequence.

File: src/tutorials-web/downloading-fraud-data.md
Comment (tonysani) [inline src/tutorials-web/downloading-fraud-data.md:1]: All good

File: src/tutorials-web/flagging-fraud-data.md
Comment (tonysani) [inline src/tutorials-web/flagging-fraud-data.md:1]: all good

File: src/tutorials-web/uploading-fraud-data.md
Comment (tonysani) [inline src/tutorials-web/uploading-fraud-data.md:1]: All good

File: src/getting-started/registering-an-account.md
Comment (tonysani) [inline src/getting-started/registering-an-account.md:1]: all good

File: src/overview/tokenomics.md
Comment (tonysani) [inline src/overview/tokenomics.md:1]: all good

File: src/overview/web-interface.md
Comment (tonysani) [inline src/overview/web-interface.md:1]: all good

**Review by tonysani — COMMENTED** [review]
Great work.  only change would be the ordering of the pages in the web UI per comment.  in terms of the screenshots, happy with suggestion.

**Review by tonysani — APPROVED** [review]

**Pull request merged**